CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 1.3.0
+* Fix table content replacement issue..
+* Tested up to WP 6.7.0.
+
+## 1.2.1
```

Comment:
I can see changes from the previous PR in here. Can we remove these changes and retain only those related to this PR?